#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

PKG = "jsk_topic_tools"

gen = ParameterGenerator()

gen.add("update_rate", double_t, 0, "Desired publish rate of throttled messages", 1.0, 0.0, 1000.0)
gen.add("approximate_sync", bool_t, 0, "Enable Approximate Synchronization", False)
gen.add("queue_size", int_t, 0, "Queue size", 100, 1, 1000)
gen.add("use_camera_info", bool_t, 0, "Synchronize camera info", False)
gen.add("force_synchronize", bool_t, 0, "Force synchronize", False)


exit(gen.generate(PKG, PKG, "SynchronizedImageThrottle"))
